pages/wizard/regions/task_step_bar.py

```diff
@@ -24,6 +24,22 @@
         'class': 'SetupType',
         'library': 'pages.wizard.rhev.setup_type',
     },
+    'Content Provider': {
```

Comment:
Please correct these to the fully qualified step name as seen in PR #36. In this case it would be 'Subscriptions.Content Provider'